onmt/modules/copy_generator.py

```diff
@@ -65,6 +64,8 @@ def __init__(self, input_size, tgt_dict):
         self.linear = nn.Linear(input_size, len(tgt_dict))
         self.linear_copy = nn.Linear(input_size, 1)
         self.tgt_dict = tgt_dict
+        self.softmax = nn.Softmax()
```

Comment:
This is fine, but you don't actually need to do this (they are parameter-less). Do you like the style better?